app/api/report.py

```diff
@@ -130,12 +84,27 @@ def add_checkin():
                     "type": "daily"
                 }})
             if len(highlight) > 0:
-                slack_msg(channel=slackChannels,
-                          msg="<@" + slack + ">!" + "\n" + "Report: " + "\n" + slackReport + "" + "\n"
-                              + "Highlight: " + highlight)
+                data = "Report: " + "\n" +slackReport + "" + "\n" + "Highlight: " + highlight
+                check_in_payload = {
+                    "user": {
+                        "email": current_user['work_email']},
+                        "data": data,
+                        "message_type" : "simple_message",
+                        "message_key": "check-in"

```

Comment:
why is message_key and type comming inside user?? this doesn't make sense. it should be outside. for user we need email, name, emp_id and phone. all are missing

Reply:
it is not sir the vs code formatter made the structure after current_user['work_email'] } this is there to close user dict ok will add name , emp_id  and phone btw phone is not there in out data structure